test/service/test_manifest.py

```diff
@@ -440,14 +440,37 @@ def test_manifest_zarr(self):
                 self.assertEqual(200, response.status_code)
                 lines = response.content.decode().splitlines()
                 file_prefix = 'output="587d74b4-1075-4bbf-b96a-4d1ede0481b2/'
+                file_location_prefix = f'url="{config.service_endpoint()}/repository/files'
                 curl_files = []
+                curl_download_urls = []
+                curl_aux_download_urls = []
                 for line in lines:
                     if line.startswith(file_prefix):
```

Comment:
The duplicated `self.assertTrue(line.endswith('"'))` threw me for a loop here. Personally I think this if/elif would benefit from an `else: pass` and possibly a comment on why nothing needs to be checked outside of these cases. But YMMV.